src/api/Reducers/propertyTree.js

```diff
@@ -59,7 +59,7 @@ const propertyOwners = (state = {}, action = {}) => {
   switch (action.type) {
     case actionTypes.addPropertyOwners: {
       const inputOwners = action.payload.propertyOwners;
-      const newState = { ...state };
+      let newState = { ...state };
```

Comment:
```suggestion
      const newState = { ...state };
```
According to linting. Even if we add things to the object further down, it does not change the reference of the object and hence it should still be const, IIRC